modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/test/java/com/liferay/dynamic/data/mapping/form/builder/internal/converter/serializer/DefaultDDMFormRuleActionSerializerTest.java

```diff
@@ -28,20 +28,6 @@
  */
 public class DefaultDDMFormRuleActionSerializerTest {
 
-	@Test
-	public void testSerializeWithEmptyTarget() {
-		DefaultDDMFormRuleAction defaultDDMFormRuleAction =
-			new DefaultDDMFormRuleAction("show", StringPool.BLANK);
-
-		DefaultDDMFormRuleActionSerializer defaultDDMFormRuleActionSerializer =
-			new DefaultDDMFormRuleActionSerializer(defaultDDMFormRuleAction);
-
-		String result = defaultDDMFormRuleActionSerializer.serialize(
-			_spiDDMFormRuleSerializerContext);
-
-		Assert.assertNull(result);
-	}
-
```

Comment:
@alinedoleron, from what I can see, this is a SF change, right? If so, please add it to a SF commit. Thanks.